v0/destinations/am/transform.js

```diff
@@ -73,6 +73,7 @@ function setPriceQuanityInPayload(message, rawPayload) {
 }
 
 function createRevenuePayload(message, rawPayload) {
+  rawPayload.productId = message.properties.product_id;
```

Comment:
This is already part of the revenue mapping JSON.

Reply:
revenue mapping JSON is being used when `revenue` & `revenue_type` is being provided in properties. In this case, when we simply provide `product_id` we are passing that info in the payload.